backend/plugin/db/mongodb/mongodb.go

```diff
@@ -397,169 +392,25 @@ func getSimpleStatementResult(data []byte) (*v1pb.QueryResult, error) {
 		return nil, err
 	}
 
-	columns, columnTypeMap, columnIndexMap, illegal := getColumns(rows)
 	result := &v1pb.QueryResult{
-		ColumnNames: columns,
-	}
-	for _, column := range columns {
-		result.ColumnTypeNames = append(result.ColumnTypeNames, columnTypeMap[column])
+		ColumnNames:     []string{"result"},
+		ColumnTypeNames: []string{"TEXT"},
 	}
 
 	for _, v := range rows {
-		d, ok := v.(bson.D)
-		if !ok || illegal {
-			r, err := json.MarshalIndent(v, "", "	")
-			if err != nil {
-				return nil, err
-			}
-			result.Rows = append(result.Rows, &v1pb.QueryRow{
-				Values: []*v1pb.RowValue{
-					{Kind: &v1pb.RowValue_StringValue{StringValue: string(r)}},
-				},
-			})
-			continue
-		}
-
-		values := make([]*v1pb.RowValue, len(columns))
-		for _, e := range d {
-			k := e.Key
-			v := e.Value
-			index := columnIndexMap[k]
-
-			switch value := v.(type) {
-			case primitive.Binary:
-				switch value.Subtype {
```

Comment:
MongoDB uses subtype to indicate the clients how to explain the binary data.
https://www.mongodb.com/zh-cn/docs/manual/reference/bson-types/#binary-data
Should we explain it in backend first before marshalling.

Related customer issue: https://linear.app/bytebase/issue/BYT-6638/better-display-mongodb-uuid-binary-data

Test Data: `db.coll.insertOne({_id: UUID(), a: 3});`

Reply:
Done. We will handle the binary uuid in the frontend